docs/user-manual/framework/supported-platforms.md

```diff
@@ -0,0 +1,32 @@
+## Supported Platforms
+
+A **supported platform** refers to a combination of hardware and operating system where the software is actively tested through continuous integration (CI). A platform is considered **supported** if all CI tests pass consistently, ensuring compatibility and reliability.  
+
+The following tables list currently supported platforms:
+
+| Hardware | OS | Reference Deployment | CI Passes |
+|------|----------|----|-----------|
+| BeagleBone Black | VxWorks 7 | [`fprime-vxworks-reference`](https://github.com/fprime-community/fprime-vxworks-reference) | TODO |
+| Raspberry Pi 5 | Linux | [`fprime-workshop-led-blinker`](https://github.com/fprime-community/fprime-workshop-led-blinker) | TODO |
```

Comment:
Technically we support the Raspberry Pi 3 per your CI definition.  There are no PI5s in the CI system.

Reply:
yet!